packages/terra-form-validation/docs/NumericValidation.md

```diff
@@ -0,0 +1,177 @@
+  if (!value) { return undefined; }
+
+  return undefined;
+};
+
+export default class Example extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+    this.submitForm = this.submitForm.bind(this);
+  }
+
+  submitForm(values) {
+    this.setState({
+      submittedValues: values,
+    });
+  }
+
+  renderForm({ handleSubmit }) {
+    return (
+      <form
+        onSubmit={handleSubmit}
+      >
+        <Field
+          name="numExample"
+          validate={validateNumber}
+        >
+          {({ input, meta }) => (
+            <InputField
+              inputId="numExample"
+              label="Numeric Example"
+              error={meta.error}
+              isInvalid={meta.error !== undefined}
+              inputAttrs={{
+                type: 'number',
+                ...input,
+              }}
+              onChange={(e) => { input.onChange(e.target.value); }}
+              value={input.value}
+              required
+            />
+          )}
+        </Field>
+        <Button text="Submit" type={Button.Opts.Types.SUBMIT} />
+      </form>
+    );
+  }
+
+  render() {
+    return (
+      <Spacer marginBottom="small">
+        <Form
+          onSubmit={this.submitForm}
+          render={this.renderForm}
+          initialValues={{ numExample: '' }}
+          validate={(values) => {
+            const errors = {};
+
+            if (!values.numExample) {
+              errors.numExample = 'Required';
+            }
+
+            return errors;
+          }}
+        />
+        {this.state.submittedValues
+          && (
+          <div>
+            <p>Form Submitted Successfully With</p>
+            <pre>{JSON.stringify(this.state.submittedValues, 0, 2)}</pre>
+          </div>
+          )
+        }
+      </Spacer>
+    );
+  }
+}
+```
+
+We have a simple number type input with validation, tied to the `<Form>` component, that checks whether we have a value for the input.
+
+We also have our `<Field>`'s validation function defined and ready to be added to, with it only having a basic check to see if there is a value.
+
+We have added browser enforcement on the inputted value by passing `type: 'number'` into `<Inputfield>`'s `inputAttrs`.
+
+Note that specifying `type: 'number'` still allows negatives and decimal points, such as `--1` or `2.0.32`. These will be placed in the field as `undefined`. With our code, we will just return a value of `Required`, via the `<Form>`'s validation function, if values are undefined.
+
+Second thing - our validation function, `validateNumber`, will be given a *string*, despite the input being of `type: 'number'`. This does not impact simple operators but is useful to know when building custom validation functions.
+
+Let's start by giving a minimum and maximum to our value. For simplicity, set a minimum to 10 and a maximum to 100.
+
+```diff
+const validateNumber = (value) => {
+  if (!value) { return undefined; }
++ if (value < 10) {
```

Comment:
nit: I feel like we should be casting `value` to a number data type here: `if (Number(value) < 10)` even though this will work without it.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/635/commits/415c46ab9a0b74893a686c38430b1d8decd708ad#diff-efb0b388a04429df36f1ce4eb79ef872R108
The other references to this code later in the example should also be fixed.